mix.exs

```diff
@@ -44,9 +44,10 @@ defmodule ExDoc.Mixfile do
       # Add other makeup lexers as optional for the executable
       {:makeup_c, ">= 0.1.0", optional: true},
       {:makeup_html, ">= 0.1.0", optional: true},
-      {:jason, "~> 1.2", only: :test},
+      {:jason, "~> 1.4"},
       {:floki, "~> 0.0", only: :test},
-      {:easyhtml, "~> 0.0", only: :test}
+      {:easyhtml, "~> 0.0", only: :test},
+      {:mdex, "~> 0.3"}
```

Comment:
Hey mdex author here. I'm happy to help, make changes upstream, or answer any question to support this PR. Thanks!

Reply:
Thank you @leandrocp for your help.  I will reach out if I need anything.